shared/structures/src/members/RegistrationWithTinyMember.ts

```diff
@@ -0,0 +1,16 @@
+import { field } from '@simonbackx/simple-encoding';
+
+import { TinyMember } from './Member.js';
+import { Registration } from './Registration.js';
+
+export class RegistrationWithTinyMember extends Registration {
```

Comment:
Snap dat je het wilt renamen, maar dit zorgt voor enkele conflicten. Zou refactors zoals een rename altijd in een aparte PR of commit proberen te doen, zodat die sneller naar main kunnen